cpu/kinetis_common/dist/antibrick.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+#
+# Anti brick check script for Freescale Kinetis MCUs.
+#
+# This script is supposed to be called from RIOTs
+# unified OpenOCD script (dist/tools/openocd/openocd.sh).
+#
+# @author       Jonas Remmert <j.remmert@phytec.de>
+# @author       Johann Fischer <j.fischer@phytec.de>
+
+ELFFILE=$1
+
+RETVAL=$(arm-none-eabi-readelf -x .fcfield $ELFFILE  | awk '/0x00000400/ {print $2 $3 $4 $5}')
```

Comment:
For extra safety it would be wise to not assume that the flash configuration section must be named .fcfield
We could do an objcopy into a hex file or binary to strip all section information and get the true memory addresses that will be loaded onto the MCU. (or just use the hex file produced by the build process directly)

`arm-none-eabi-objdump --start-address=0x400 --stop-address=0x410 ${HEXFILE} -s | awk -F' ' '/^ 0400 / { print $2 $3 $4 $5; }'` does the same as the above line but works on a hex file.